ide-config/src/main/resources/cdi.importorder

```diff
@@ -0,0 +1,6 @@
+#Organize Import Order
+0=java
+1=javax
+2=jakarta
+3=org
+4=com
```

Comment:
Is this file ever used?

Reply:
No, this is only present for importing into IDEs so that the formatting is identical to that of `impsort` plugin.
If we think that's not needed, we can safely remove this.

Reply:
I have no strong feeling about this, if you want me to remove this, just let me know